### DISCUSSION TITLE
Expand to framework agnostic

### DISCUSSION THREAD

**Discussion by Brian-McBride** [avkonst/react-use-state-x#385] [General]
Hookstate is great. 
Sadly, my teams have slowly migrated away as we are forced to work within multiple frameworks. Some newer ones, such as Astro, are very cool. 

And such, it would be pretty awesome to see Hookstate abstracted in such a way that it works for all frameworks where `useHookstate` becomes a framework-specific subscriber.

**Comment by avkonst** [comment]
1. It is a great idea
2. I had integration done with vue in the past, but this work was cancelled as a user lost interest in this.
3. It is hard to make a reality as I work on this at free time and this time availability is shrinking more and more. It might only happen if someone joins the forces to make it a reality
